app/controllers/application_controller.rb

```diff
@@ -1065,4 +1068,20 @@ def link_preload
   def spa_boot_request?
     request.get? && !(request.format && request.format.json?) && !request.xhr?
   end
+
+  def load_font_map
+    DiscourseFonts
+      .fonts
+      .reduce({}) do |font_map_hash, font|
+        if font[:variants]
+          font_map_hash[font[:key]] = font[:variants].map do |v|
+            {
+              url: "#{Discourse.base_url}/fonts/#{v[:filename]}?v=#{DiscourseFonts::VERSION}",
+              weight: v[:weight],
+            }
+          end
+        end
+        font_map_hash
+      end
```

Comment:
Warning, I'm codegolfing here 😄 

```suggestion
      .each_with_object({}) do |font, font_map|
        next if !font[:variants]
        font_map[font[:key]] = font[:variants].map do |v|
          {
            url: "#{Discourse.base_url}/fonts/#{v[:filename]}?v=#{DiscourseFonts::VERSION}",
            weight: v[:weight],
          }
        end
      end
```

Reply:
Nothing wrong with a _little_ golf, just some putt-putt 😆